src/components/helpers/choices.js

```diff
@@ -249,12 +249,15 @@ export default createReactClass({
       const rect = node.getBoundingClientRect();
       const top = rect.top;
       const windowHeight = window.innerHeight;
-      const height = windowHeight - top;
-      if (height !== this.state.maxHeight) {
+      const maxHeightRatio = this.props.config.getChoicesMaxHeightRatio();
+      const clampedRatio = Math.min(Math.abs(maxHeightRatio), 1);
```

Comment:
This will blow up if `config.getChoicesMaxHeightRatio` is not defined, and it won't be by default. I think we should add a default implementation to the [default formatic config](https://github.com/zapier/formatic/blob/master/src/default-config.js)

```js
  getChoicesMaxHeight {
    return 1;
  }
```

The default formatic config also kind of serves as documentation, letting you know what can be customized.